src/BizHawk.Client.Common/RomLoader.cs

```diff
@@ -481,6 +482,7 @@ private void LoadOther(CoreComm nextComm, HawkFile file, string forcedCoreName,
 						RomData = rom.RomData,
 						FileData = rom.FileData,
 						Extension = rom.Extension,
+						RomPath = file.Directory,

```

Comment:
Don't call it `RomPath` if it contains the parent directory

Reply:
What else would you suggest? `RomDirectory`?

Reply:
That's fine. Or use `RomPath` but include the filename, it seems the callsite is reconstructing it anyway.